src/main/java/hudson/plugins/jira/JiraRestService.java

```diff
@@ -94,15 +94,23 @@
     
     private final int timeout;
 
+    private final boolean sendAssigneeAsName;
+
     @Deprecated
     public JiraRestService(URI uri, ExtendedJiraRestClient jiraRestClient, String username, String password) {
     	this(uri, jiraRestClient, username, password, JiraSite.DEFAULT_TIMEOUT);
     }
 
+    @Deprecated
     public JiraRestService(URI uri, ExtendedJiraRestClient jiraRestClient, String username, String password, int timeout) {
+        this(uri, jiraRestClient, username, password, timeout, false);
+    }
+
+    public JiraRestService(URI uri, ExtendedJiraRestClient jiraRestClient, String username, String password, int timeout, boolean sendAssigneeAsName) {
```

Comment:
How about using a builder pattern here, so not changing the current constructor and setting it later? (see [JiraSiteBuilder](https://github.com/jenkinsci/jira-plugin/blob/30999c946ded4581fecfe9e2719b6edf0ffcbc60/src/main/java/hudson/plugins/jira/JiraSite.java#L1236) for example) ?

Reply:
Sure.